docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -116,7 +116,7 @@ oc wait --for condition=Ready pod/mariadb-copy-data --timeout=30s
 
 .Procedure
 
-. Check that the Galera database cluster members are online and synced:
+. Check that the Source galera database cluster members are online and synced:
```

Comment:
I think this should be capitalized differently: "source Galera". It's a small thing but now that these docs directly translate to customer docs, we need to care about such stuff :).

Reply:
updated in next update